views/forms/referral_form.php

```diff
@@ -0,0 +1,744 @@
+    $office_means = !empty($_POST['office_means']) ? trim($_POST['office_means']) : "";
+    $office_initial_date = !empty($_POST['office_initial_date']) ? trim($_POST['office_initial_date']) : NULL;
+    $office_time = !empty($_POST['office_time']) ? trim($_POST['office_time']) : "";
+    $office_location = !empty($_POST['office_location']) ? trim($_POST['office_location']) : "";
+    $comments = !empty($_POST['comments']) ? trim($_POST['comments']) : "";
+
+
+
+
+    // TODO: stored procedure
+
+    /*              This is stored procedure work that is being commented out for now so we can at least have our forms in develop for the rest of the class to see.
+
+    $con = pg_connect('host=10.11.12.33 port=5432 user=postgres password=@lgozzineIsTheBest dbname=PEP_DB');
+    pg_query($con, 'BEGIN;');
+    $result = pg_query_params($con,
+        'SELECT addAgencyReferral(
+          fName := $1::TEXT,
+          lName := $2::TEXT,
+          mInit := $3::VARCHAR,
+          dob :=  $4::DATE,
+          houseNum := $5::INTEGER,
+          streetAddress := $6::TEXT,
+          apartmentInfo := $7::TEXT,
+          zipCode := $8::INTEGER,
+          city := $9::TEXT,
+          state := $10::STATES,
+          referralReason := $11::TEXT,
+          hasAgencyConsentForm := $12::BOOLEAN,
+          referringAgency := $13::TEXT,
+          referringAgencyDate := $14::DATE,
+          additionalInfo := $15::TEXT,
+          hasSpecialNeeds :=$16::BOOLEAN,
+          hasSubstanceAbuseHistory :=$17::BOOLEAN,
+          hasInvolvementCPS :=$18::BOOLEAN,
+          isPregnant :=$19::BOOLEAN,
+          hasIQDoc := $20::BOOLEAN,
+          mentalHealthIssue := $21::BOOLEAN,
+          hasDomesticViolenceHistory := $22::BOOLEAN,
+          childrenLiveWithIndividual := $23::BOOLEAN,
+          dateFirstContact := $24::DATE,
+          meansOfContact := $25::TEXT,
+          dateOfInitialMeeting := $26::DATE,
+          location := $27::TEXT,
+          comments := $28::TEXT,
+          eID := $29::INTEGER)',
+        array(
+            '$pers_firstname',
+            '$pers_lastname',
+            '$pers_middlein',
+            $pers_dob,
+            $pers_address_num,
+            '$pers_address_street',
+            '$pers_apartment',
+            $pers_zip,
+            '$pers_city',
+            $pers_state,
+            '$pers_reason',
+            $chkSigned,
+            '$ref_party',
+            $ref_date,
+            '$additional_info',
+            $chkSpecialEd,
+            $chkSubAbuse,
+            $chkCPS,
+            $chkPreg,
+            $chkIQ,
+            $chkMental,
+            $chkViolence,
+            $chkReside,
+            $office_contact_date,
+            '$office_means',
+            $office_initial_date,
+            '$office_location',
+            '$comments',
+            1)); */
+
+
+    header("Location: /form-success?type=$form_type");
```

Comment:
Don't use PHP's GET url forms with the ?key=value stuff. The best way to pass along the form type here would be to use a session variable. Since you can control where the user is going with `header()` you can unset the session variable after.

So here do something like:
```php
$_SESSION['form-type'] = $form_type;
header("Location: /form-success");
die();
```